src/main/java/com/management/HumanResources/model/EmployeeTime.java

```diff
@@ -1,14 +1,32 @@
 package com.management.HumanResources.model;
 
 import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import org.json.JSONObject;
+
 import lombok.Data;
 
 @Data
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class EmployeeTime {
 
     private long employeeId;
     private double hoursRemaining;
     private double totalHours;
     private List<TimeOff> timeOffs;
     private String[] availability;
+    private String csvTimeOff;
+    private String csvAvailability;
+
+    public String toDatabaseString() {
```

Comment:
I think `toJson' would be more clear of what this method does.

Reply:
Should we maybe have a service that converts objects into JSON?

Reply:
I see your point but that might not be necessary. The objects are already serialized to Json in the webclient methods. I just have this method here so that it picks up the correct csv formatted values.